packages/@sanity/desk-tool/src/pane/DocumentsPane.js

```diff
@@ -121,6 +121,18 @@ export default withRouterHOC(class DocumentsPane extends React.PureComponent {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.selectedType !== this.props.selectedType) {
+      const settings = readSettings()
+      this.setState({
+        settings: (settings && settings[nextProps.selectedType]) || {
+          listLayout: 'default',
+          ordering: DEFAULT_SELECTED_ORDERING_OPTION
+        }
```

Comment:
The default setting will now be duplicated. Could you extract `{listLayout: 'default', ordering: DEFAULT_SELECTED_ORDERING_OPTION}` into a constant named `DEFAULT_SETTINGS`, and also point to this in the constructor?

Reply:
oh. Seems like I had this branch from earlier with changes. Changes is from Dec 12. This commit is not a part of this PR. Will remove.